examples/post.v3.LR.historical.zppy_v3.cfg

```diff
@@ -0,0 +1,289 @@
+  # "streamflow"
+  extra_vars = 'areatotal2'
+  frequency = "monthly"
+  input_files = "mosart.h0"
+  input_subdir = "archive/rof/hist"
+  mapping_file = ""
+  vars = "RIVER_DISCHARGE_OVER_LAND_LIQ"
+
+  [[ atm_monthly_glb ]]
+  # This subtask is a dependency for the global_time_series task.
+  frequency = "monthly"
+  input_files = "eam.h0"
+  input_subdir = "archive/atm/hist"
+  mapping_file = "glb"
+
+  [[ lnd_monthly_glb ]]
+  # This subtask is a dependency for the global_time_series task.
+  frequency = "monthly"
+  input_files = "elm.h0"
+  input_subdir = "archive/lnd/hist"
+  mapping_file = "glb"
+  # vars = "FSH,RH2M,LAISHA,LAISUN,QINTR,QOVER,QRUNOFF,QSOIL,QVEGE,QVEGT,SOILWATER_10CM,TSA,H2OSNO,TOTLITC,CWDC,SOIL1C,SOIL2C,SOIL3C,SOIL4C,WOOD_HARVESTC,TOTVEGC,NBP,GPP,AR,HR"
+  vars = "" # This will tell zppy to use all available variables
+
+[e3sm_to_cmip]
+# This is a new task in zppy v3.0.0!
+# This task replaces the `ts_fmt = "cmip"` functionality previously found in the `ts` task.
+active = True
+frequency = "monthly"
+# An alternative to setting `ts_grid` here is setting `ts_atm_grid` and `ts_land_grid` in the respective subtasks below.
+ts_grid = "180x360_aave"
+ts_num_years = 30
+walltime = "00:50:00"
+years = "1985:2014:30"
+
+  [[ atm_monthly_180x360_aave ]]
+  # This subtask is a dependency for the ilamb task.
+  # This subtask depends on the ts task's atm_monthly_180x360_aave subtask.
+  # Notice this subtask name matches a subtask in the `ts` task.
+  # If it did not, then the `ts_atm_subsection` parameter would be required here to tell zppy which subtask to use.
+  input_files = "eam.h0"
+
+  [[ land_monthly ]]
+  # This subtask is a dependency for the ilamb task.
+  # This subtask depends on the ts task's land_monthly subtask.
+  # Notice this subtask name matches a subtask in the `ts` task.
+  # If it did not, then the `ts_land_subsection` parameter would be required here to tell zppy which subtask to use.
+  input_files = "elm.h0"
+
+[tc_analysis]
+# This subtask is a dependency for the e3sm_diags task's atm_monthly_180x360_aave subtask.
+# 1 DEPRECATED PARAMETER in zppy v3.0.0:
+# scratch
+active = True
+walltime = "02:00:00"
+years = "1985:2014:30",
+
+[e3sm_diags]
+active = True
+multiprocessing = True
+num_workers = 8
+ref_final_yr = 2014
+ref_start_yr = 1985
+ts_num_years = 30
+walltime = "4:00:00"
+years = "1985:2014:30",
+
+  [[ atm_monthly_180x360_aave ]]
+  # NOTE: If you want to use the latest development environment of e3sm_diags,
+  # you can do something like the following:
+  # environment_commands = "source /home/ac.zhang40/y/etc/profile.d/conda.sh; conda activate e3sm_diags_dev"
+  # `e3sm_diags` is largely driven by which e3sm_diags sets are requested:
+  sets="lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","annual_cycle_zonal_mean","qbo","diurnal_cycle","zonal_mean_2d_stratosphere","aerosol_aeronet","tropical_subseasonal","tc_analysis", "tropical_subseasonal",
```

Comment:
Fix alignment

Reply:
@golaz I think that is just how it looks in the diff view. If you look at the file in https://github.com/E3SM-Project/zppy/blob/0917f59e9d48a2f7d246ed99b50eae0b352ed9a2/examples/post.v3.LR.historical.zppy_v3.cfg, it is aligned.